test/fixtures/view_generator/terraform.tfvars

```diff
@@ -0,0 +1,37 @@
+default_table_expiration_ms = 3600000
```

Comment:
Test fixtures should not require a tfvars file. Just hardcode most of these values or set them as defaults on the variable.